### ISSUE TITLE
Loss functions and training script?

### ISSUE THREAD

**Issue by clvnmng** [egurapha/IgVAE#2]
Hi,

Thanks for awesome publication. I couldn't find your loss functions and training script? Can you please provide more details thanks.

**Comment by egurapha** [comment]
Hi @clvnmng , thanks for your comment! We plan to release training code in the future for some new models that we're working on (upgraded versions of this model using newer modules), but likely not for this one. Stay tuned for releases in the future! All the best, R.

**Comment by clvnmng** [comment]
Can you at least please fill up the details: loss functions, codes for setting up torsion angles? thanks
